wdl4s/0.3/api/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html >
```

Comment:
Do not bring these HTML files over into `develop`.

There were only ever supposed to be in the [`gh-pages`](https://broadinstitute.atlassian.net/wiki/spaces/DSDEEPB/pages/20480292/Release+WDL+steps#ReleaseWDLsteps-Releasingwdl4s) branch, but for some reason they leaked into wdl4s develop. Leave them behind.

Reply:
Agreed. I kept meaning to remove them but kept being lazy. Laziness paid off here :)

Reply:
@kshakir more than happy to oblige. 😄 
@geoffjentry it usually does.